tests/kafkatest/tests/client/run_client_compatibility_test.py

```diff
@@ -45,15 +45,15 @@ def run_command(node, cmd, ssh_log_file):
             raise e
 
 
-class RunCompatibilityTest(Test):
+class RunClientCompatibilityTest(Test):
```

Comment:
Nit: I think I'd just call this `ClientCompatibilityTest`. The fact that it runs the Java tool to do that is a bit of an implementation detail, isn't it?